package.json

```diff
@@ -44,7 +44,6 @@
   },
   "dependencies": {
     "angular": "^1.6.6",
-    "angular-ckeditor": "lemonde/angular-ckeditor#v1.0.3",
```

Comment:
Why was this removed?

Reply:
Isn't this redundant. We pull angular-ckeditor from the bower.js file.

Reply:
Bower and NPM are two different ways to pull in this module. It's redundant because we provide people a choice which one they want to use.